Classes/Event/AuthenticationProcessMappingEvent.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the TYPO3 CMS project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
+namespace Causal\Oidc\Event;
+
+use Psr\Http\Message\RequestInterface;
+
+final class AuthenticationProcessMappingEvent
+{
+    public function __construct(
+        public readonly RequestInterface $request,
+        public readonly string $databaseTable,
+        public readonly array $existingUser,
+        public readonly array $idPdata,
```

Comment:
This property seems to be called differently for different events. 

* [resourceInfo](https://github.com/xperseguers/t3ext-oidc/blob/master/Classes/Event/AuthenticationFetchUserEvent.php#L43)
* [resource](https://github.com/xperseguers/t3ext-oidc/blob/master/Classes/Event/AuthenticationGetUserGroupsEvent.php#L30)
* [resourceOwner](https://github.com/xperseguers/t3ext-oidc/blob/master/Classes/Event/ModifyResourceOwnerEvent.php)
* [oidcResourceOwner](https://github.com/xperseguers/t3ext-oidc/blob/master/Classes/Event/ModifyUserEvent.php)

and now the new `idPdata` :)

Reply:
This is a mess, I know.
I'd settle on one name for this kind of data and use it here now for the event.
The cleanup in the other places should be a different patch then, agree?

Reply:
I'd aim for `$oidcResourceOwner` now. What do you think?

Reply:
Going with `resourceOwner` then.